mutacc/cli/importing.py

```diff
@@ -0,0 +1,56 @@
+import json
+import sys, os
+
+import click
+
+from mutacc.utils.hk_call import find_files
+
+@click.command('import')
+@click.option('--bam', multiple = True, type = click.Path(exists = True), help = 'path to .bam file')
+@click.option('--fastq', multiple = True, type = click.Path(exists = True), help = 'path to .fastq file')
+@click.option('--variant', help = 'Variant data as JSON string, or path to .json file')
+@click.option('--case', help = 'Metadata for the case as JSON string or path to .json file')
+@click.pass_context
+def importing(context,bam,fastq,variant,case):
+
+    """
+        Import cases to the database
+    """
+            
+    case = parse_JSON(case)
+    variant = parse_JSON(variant)
+
+    for path in bam:
+        click.echo(path)
+    
+    click.echo(case)
+    click.echo(variant)
+
+def parse_JSON(json_format):
+
+    if json_format[0] == '~':
+
+        json_format = os.path.expanduser(json_format)
```

Comment:
I hope you are using python version >=3.6. In that case you might want to have a look at the [path lib](https://docs.python.org/3/library/pathlib.html)